### ISSUE TITLE
xdotool parameter should be -v instead of version in LinuxUtil.java - Segmentation fault

### ISSUE THREAD

**Issue by pla1** [RaiMan/SikuliX-2014#253]
Hi, 

I think `CommandLine.parse("xdotool version")` should be `CommandLine.parse("xdotool -v")` in the `checkFeatureAvailability` method of `LinuxUtil.java`.

On Ubuntu 14.04 and Ubuntu 16.04 `xdotool version` fails with a segmentation fault. 

It is a bug in `xdotool` that any unknown parameter that is passed causes a segfault. See my tests below: 
```
htplainf@ub-mguibot:~$ cat /etc/issue
Ubuntu 16.04.2 LTS \n \l
htplainf@ub-mguibot:~$ xdotool -v
xdotool version 3.20150503.1
htplainf@ub-mguibot:~$ xdotool version
Segmentation fault (core dumped)
htplainf@ub-mguibot:~$ xdotool any_unknown_param_causes_segfault
Segmentation fault (core dumped)
htplainf@ub-mguibot:~$ 
```
Thank-you!

PLA

**Comment by RaiMan** [comment]
Thanks for finding:

made a bug: https://bugs.launchpad.net/sikuli/+bug/1687263

... and will fix it in version 1.1.2

**Comment by RaiMan** [comment]
fixed per today: see
https://raiman.github.io/SikuliX-2014/nightly.html
